autocomplete/bash_autocomplete

```diff
@@ -3,14 +3,19 @@
 : ${PROG:=$(basename ${BASH_SOURCE})}
 
 _cli_bash_autocomplete() {
+  if [[ "${COMP_WORDS[@]:0:$COMP_CWORD}" != "source" ]]; then
```

Comment:
Some shellcheck hint about this: 
`SC2199: Arrays implicitly concatenate in [[ ]]. Use a loop (or explicit * instead of @).`
See https://github.com/koalaman/shellcheck/wiki/SC2199

Reply:
replace with exact element match in array